### DISCUSSION TITLE
Are ForkLift Sets now just Favorites?

### DISCUSSION THREAD

**Discussion by chellman** [ttscoff/bunch-docs#292] [Q&A] [answered]
Hello! New Bunch user who's been using ForkLift for a little while longer.  The [docs mention a ForkLift feature called Sets](https://bunchapp.co/docs/integration/forklift/) for saving window configurations.  I don't see anything called that in ForkLift 3.5.8.

It looks like there are Favorites, which populate the sidebar as well. I've made a new Favorites group called Workspaces and dropped my window/tab sets in there so my sidebar is a little less crazy. I'm guessing this is the expected way to do this, but I wanted to check.

Thank you!

**Comment by ttscoff** [comment, marked as answer]
Yeah, it's just a saved Workspace. Maybe a holdover from a previous era, 
but when I save a workspace under Favorites, I have the option to add to 
a group called Sets. Those are probably free-form group names, so I may 
have just named it that and then forgot that wasn't a default 
nomenclature. Will update the docs accordingly.

**Reply by chellman** [reply]
Yeah, must be.  I switched to ForkLift from Path Finder a year or two ago, and I seem to recall that PF might have called this feature Sets.  Maybe you had a similar journey to mine (i.e. "holy smoke Path Finder is becoming a mess"), and carried the terminology through.

Thanks for taking a look.
